src/land/afwa_snod2ioda.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+#
+# (C) Copyright 2020 UCAR
```

Comment:
Change Copyright to be NOAA NWS NCEP EMC, not UCAR

Reply:
Changed to NOAA/NWS/NCEP/EMC.